test/inference/gibbs_conditional.jl

```diff
@@ -0,0 +1,144 @@
+using Random, Turing, Test
+
+dir = splitdir(splitdir(pathof(Turing))[1])[1]
+include(dir*"/test/test_utils/AllUtils.jl")
+
+
+@turing_testset "gibbs conditionals" begin
+    let N = 1000,
+        (α_0, θ_0) = (2.0, inv(3.0)),
+        λ_true = rand(Gamma(α_0, θ_0)),
+        σ_true = √(1 / λ_true),
+        m_true = rand(Normal(0, σ_true)),
+        x = rand(Normal(m_true, σ_true), N)
+        
+        @model inverse_gdemo(x) = begin
+            λ ~ Gamma(α_0, θ_0)
+            m ~ Normal(0, sqrt(1 / λ))
+            x .~ Normal(m, sqrt(1 / λ))
```

Comment:
You have to use, e.g.,
```suggestion
            @. x ~ $(Normal(m, sqrt(1 / λ)))
```
or
```suggestion
            (.~)(x, Normal(m, sqrt(1 / λ)))
```
since `.~` can't be parsed by Julia 1.0 (see, e.g., https://github.com/TuringLang/DynamicPPL.jl/blob/06e7fe0cd08a22e728306a9444cf45d0e3ede690/test/compiler.jl#L440-L447).